epochStart/metachain/systemSCs.go

```diff
@@ -179,6 +203,14 @@ func (s *systemSCProcessor) selectNodesFromAuctionList(validatorsInfoMap state.S
 	return nil
 }
 
+// TODO: Move this in elrond-go-core
+func safeSub(a, b uint32) (uint32, error) {
```

Comment:
we have this for Uint64

Reply:
Right, but I didn't want to cast to uint64, but have a separate function for it. Will move it in elrond-go-core in a separate task